scripts/preprocess-health.py

```
@@ -0,0 +1,425 @@
+from datetime import datetime, timezone
+
+# Windows defaults stdout to cp1252 under Python 3.8 — `→` and other
+# Unicode glyphs in our log lines crash the run. Force UTF-8 so the
+# script works the same on macOS / Linux / Windows. Safe no-op on
+# already-UTF-8 streams.
+if hasattr(sys.stdout, 'reconfigure'):
+    try:
+        sys.stdout.reconfigure(encoding='utf-8')
+        sys.stderr.reconfigure(encoding='utf-8')
+    except Exception:  # pragma: no cover — older streams without reconfigure
+        pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)